index.html

```diff
@@ -153,35 +237,49 @@ <h3 id="h3-lang">
                     </ul>
                 </div>
             </div>
+        </div>
         </div>      
     </section>
     <div class="project-1" id="about-me-section-4">
         <div class="am-contact-me-main-grid">
          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
-        <div>
+        <div class="inputs-form">
          <form class="contact" action="https://formspree.io/f/mqkoyzva" method="post">
-           <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+           <input type="text" name="name" placeholder="Full Name" maxlength="30" required>
            <input type="email" name="email" placeholder="Email address" required>
-           <textarea name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+           <textarea class="txtS" name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
            <button type="submit" name="submit" id="contact-button">Get in touch</button>
         </form>
+        </div>
+        <div class="inputs-form2">
+            <form class="contact" action="https://formspree.io/f/mqkoyzva" method="post">
+                <div class="first-last-name">
+                <input class="first-name" type="text" name="name" placeholder="First Name" maxlength="15" required>
+                <input class="last-name" type="text" name="name" placeholder="Last Name " maxlength="15" required>
+                </div>
+                <input type="email" name="email" placeholder="Email address" required>
+                <textarea class="txtS" name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+                <button type="submit" name="submit" id="contact-button">Get in touch</button>
+             </form>
```

Comment:
- [ ] Kindly update the width of this contact form so it sticks to the Figma design 👍 

![Capture 4](https://user-images.githubusercontent.com/80458199/228807857-ff1a8fcf-b6dc-435e-a0bf-369787eb07af.PNG)